src/convertToObject.js

```diff
@@ -6,7 +6,15 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString.split(';').reduce(addStyle, {});
+}
+
+function addStyle(styles, string) {
+  const style = string.split(':').map((item) => item.trim());
+
+  styles[style[0]] = style[1];
```

Comment:
Ensure that `style[0]` and `style[1]` are valid before assigning them to the `styles` object. This will prevent potential errors if the input string is malformed or contains empty style definitions.